### PR TITLE
Default dump path for kind deployment

### PR DESCRIPTION
**Pull request by BenTheElder**
cc @neolit123 this should be easier than https://github.com/kubernetes/test-infra/pull/11336

### CODE REVIEW

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ periodics:
       - --down
       # specific e2e test args
       # TODO(bentheelder): num-nodes in particular is brittle. it would be better for kubetest to handle this
-      - --test_args=--ginkgo.focus=\[Conformance\] --ginkgo.skip=\[Serial\]|Alpha|Kubectl|\[(Disruptive|Feature:[^\]]+|Flaky)\] --disable-log-dump=true --num-nodes=3
+      - --test_args=--ginkgo.focus=\[Conformance\] --ginkgo.skip=\[Serial\]|Alpha|Kubectl|\[(Disruptive|Feature:[^\]]+|Flaky)\] --num-nodes=3
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml]: `--disable-log-dump=true` should be set automatically if --dump is set

**Comment by BenTheElder** [comment]
that worked :-)
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-kind-master/224
/hold cancel

**Comment by neolit123** [comment]
was just checking - awesome!!!!! 
and thank's a lot, of course.

**Pull request merged**